setup.cfg

```diff
@@ -0,0 +1,119 @@
+# This file is used to configure your project.
+# Read more about the various options under:
+# http://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files
+
+[metadata]
+name = ssht_numba
+description = Add a short description here!
```

Comment:
We should add a description for the package.

Reply:
Done. And removed my name as Author... that was automatic from pyscaffold